src/main/java/iudx/rs/proxy/apiserver/ApiServerVerticle.java

```diff
@@ -110,6 +122,12 @@ public void start() throws Exception {
     attachCORSHandlers(router);
     attachDefaultResponses(router);
     router.route().handler(BodyHandler.create());
+    router.route().handler(new ConsentLogRequestHandler(true));
```

Comment:
should the constructor parameter be configurable here?

Reply:
Yes, It's configurable. you can read the `isAdexDeployed` property from the config and pass it as a constructor parameter.
Currently, In PR it is just kept as true, that needs to be changed before emerging PR.